app/app.go

```diff
@@ -414,9 +412,10 @@ func NewCanto(
 	app.AccountKeeper = authkeeper.NewAccountKeeper(
 		appCodec, keys[authtypes.StoreKey], app.GetSubspace(authtypes.ModuleName), ethermint.ProtoAccount, maccPerms,
 	)
-	app.BankKeeper = bankkeeper.NewBaseKeeper(
+	baseKeeper := bankkeeper.NewBaseKeeper(
 		appCodec, keys[banktypes.StoreKey], app.AccountKeeper, app.GetSubspace(banktypes.ModuleName), app.BlockedAddrs(),
 	)
+	app.BankKeeper = baseKeeper
```

Comment:
Is there a reason or intention to do this?

Reply:
I think we can delete it. Done in https://github.com/b-harvest/Canto/commit/781a833530b010dc56fa834b259d284468ba2632.